src/main/java/model/piece/moveRule/MoveRule.java

```diff
@@ -0,0 +1,11 @@
+package model.piece.moveRule;
+
+import model.piece.position.Position;
+
+import java.util.List;
+
+public interface MoveRule {
+
+    List<Position> calculateRouteToDestination(final Position startPosition, final Position destination);
+
+}
```

Comment:
```suggestion
    List<Position> calculateRouteToDestination(final Position startPosition, final Position destination);
}
```
이런 불필요 개행도 전반적으로 확인해주세요~

Reply:
넵!